docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -386,7 +386,7 @@ if [ -f core.zst ]; then
     CORE_LINK='<a href="core.zst">core.zst</a>'
 fi
 
-rg --text -F '<Fatal>' server.log > fatal.log ||:
+sed -n '/<Fatal>/,/^$/p' s.log | rg "(^[^202])|<Fatal>" server.log > fatal.log ||:
```

Comment:
Ok, but 202 is funny.

Reply:
It's wrong too, it's not matching `202` but the first character only

Reply:
Should be ok now. Much cleaner using `awk` and an OR